src/umpire/Umpire.hpp

```diff
@@ -18,6 +19,8 @@
 #include "umpire/util/MPI.hpp"
 #include "umpire/util/io.hpp"
 
+using Resource = camp::resources::Resource;
```

Comment:
This pollutes the global namespace and should be removed.

Reply:
Done